keps/sig-api-machinery/00xx-admission-webhooks-to-ga.md

```diff
@@ -668,7 +668,7 @@ const (
 // ValidatingWebhookConfiguration describes the configuration of and admission webhook that accept or reject and object without changing it.
 type ValidatingWebhookConfiguration struct {
      metav1.TypeMeta `json:",inline"`
-     // Standard object metadata; More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata.
+     // Standard object metadata; More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#metadata.
```

Comment:
We really want to keep these as git.k8s.io, so here `https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata`

Same on all of these

Reply:
OK, I'll change it